types/prettier/standalone.d.ts

```diff
@@ -0,0 +1,43 @@
+import { CursorOptions, CursorResult, Options, Plugin } from './';
+
+type Omit<GType, GKeys extends keyof GType> = Pick<
+    GType,
+    Exclude<keyof GType, GKeys>
+>;
+
+type MakeRequired<GType extends {}, GKeys extends keyof GType> = Omit<
+    GType,
+    GKeys
+> &
+    { [P in GKeys]-?: GType[P] };
+
+type MR<GType extends { parser?: any }> = MakeRequired<GType, 'parser'>;
```

Comment:
The `parser` option is not required here, it's same as the one used in `index.d.ts`, which defaults to `babylon` (`babel`).